man/sd_notify.xml

```diff
@@ -202,6 +202,14 @@
         watchdog is enabled. </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term>EXTEND_TIMEOUT=1</term>
+
+        <listitem><para>Tells the service manager to set the startup or shutdown service timeout be the watchdog time
```

Comment:
Please drop the "the" in "the watchdog time".

Reply:
"… to extend the start or stop service timeout by the watchdog timeout."